Devscord.DiscordFramework/Commons/Extensions/StringBuilderExtensions.cs

```diff
@@ -1,26 +1,33 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
 using System.Text;
 
 namespace Devscord.DiscordFramework.Commons.Extensions
 {
     public static class StringBuilderExtensions
     {
-        public static StringBuilder PrintManyLines(this StringBuilder builder, string header, string[] lines, bool contentStyleBox = true, string formatName = "")
+        public static StringBuilder PrintManyLines(this StringBuilder builder, string header, string[] lines, bool contentStyleBox = true, bool spacesBetweenLines = false, string formatName = "")
         {
             builder.AppendLine(header);
-            PrintManyLines(builder, lines, contentStyleBox, formatName);
+            PrintManyLines(builder, lines, contentStyleBox, spacesBetweenLines, formatName);
             return builder;
         }
 
-        public static StringBuilder PrintManyLines(this StringBuilder builder, string[] lines, bool contentStyleBox = true, string formatName = "")
+        public static StringBuilder PrintManyLines(this StringBuilder builder, string[] lines, bool contentStyleBox = true, bool spacesBetweenLines = false, string formatName = "")
         {
-            if(contentStyleBox)
+            //can fix message if response looks like "test {outputFromPrintManyLines}"
+            builder.AppendLine();
+            if (contentStyleBox)
             {
                 builder.AppendLine($"```{formatName}");
             }
             foreach (var line in lines)
             {
                 builder.AppendLine(line);
+                if (spacesBetweenLines && line.GetHashCode() != lines.Last().GetHashCode())
+                {
+                    builder.AppendLine();
```

Comment:
Co jak wszystkie _lines_ będą takie same? Nie będą wtedy również Hash'e takie same?

Reply:
hash to ID referencji
nie będzie takiej sytuacji

Reply:
no chyba że ktoś wrzuci chamskiego foreach z tym samym obiektem, ale czegoś takiego nie chcemy nigdzie i to nie przejdzie code review

Reply:
w sensie foreach generujący pętle, która ma dużo razy ten sam obiekt